frontend/src/apis/sales/SalesAPI.tsx

```diff
@@ -43,6 +51,32 @@ export interface DeleteSRReq {
   id: string;
 }
 
+// addSalesReconciliation
+export interface AddSRReq {
+  date: string;
+  sales: APISRSaleRow[];
+}
+
+// modifySalesReconciliation
+export interface ModifySRReq extends AddSRReq {
+  id: string;
+}
+
+// salesReconciliationsCSVImport
+export interface SRCSVImportReq {
+  file: File;
+}
+
+export interface APISaleCSVImportRow extends APISRSaleRow {
```

Comment:
This format actually changed, take a look at the other files. I decided not to make the CSV row extend anything for BB/PO. All of these endpoints need to be tested to ensure they work as expected, or else there's really no point in merging in this code